man/pdfpc.in

```diff
@@ -124,6 +124,10 @@ Go to the last slide
 .B Backspace
 Go back in history. Note that history is defined by "jump" commands, not by normal slide movement.
 .TP
+.B j / k
```

Comment:
`j` is down, and `k` is up in VIM. But you're right, I was wondering whether this'd be a good idea myself. Swapped.